packages/liquidator/src/liquidationStrategy.js

```diff
@@ -142,46 +129,44 @@ module.exports = (
     assert(currentBlockTime >= 0, "requires currentBlockTime");
     assert(position, "requires position");
 
-    // whale fund enabled, check if withdraw liveness can be extended with a min liquidation
-    if (whaleDefenseFundWei && toBN(whaleDefenseFundWei).gt(toBN("0"))) {
-      // we should only liquidate the minimum here to extend withdraw
-      if (
-        shouldLiquidateMinimum({
-          position,
-          syntheticTokenBalance,
-          currentBlockTime
-        })
-      ) {
-        // we are using the WDF to liquidate. We need to alert user
-        emit("info", {
-          message:
-            "Liquidator bot is extending withdraw deadline, funds may need to be added to liquidate full position",
-          position: position,
-          maxLiquidationPrice: maxCollateralPerToken.toString(),
-          syntheticTokenBalance: syntheticTokenBalance.toString(),
-          maxTokensToLiquidateWei: maxTokensToLiquidateWei ? maxTokensToLiquidateWei.toString() : null,
-          whaleDefenseFundWei: whaleDefenseFundWei.toString(),
-          currentBlockTime,
-          ...logInfo
-        });
-        // this assume empMinSponsorSize will always be <= maxTokensToLiquidate
-        return createLiquidationParams({
-          sponsor: position.sponsor,
-          // Only liquidating the minimum size to extend withdraw
-          tokensToLiquidate: minSponsorSize,
-          currentBlockTime,
-          maxCollateralPerToken
-        });
-      }
-    }
+    let tokensToLiquidate = toBN(0);
 
-    // WDF strategy not activated, calculate tokens to liquidate respecting all constraints
-    const tokensToLiquidate = calculateTokensToLiquidate({
-      syntheticTokenBalance,
-      positionTokens: position.numTokens,
-      maxTokensToLiquidateWei,
-      withdrawRequestPending: hasWithdrawRequestPending({ position, currentBlockTime })
-    });
+    if (!wdfActive || !hasWithdrawRequestPending({ position, currentBlockTime })) {
```

Comment:
Hopefully this if-else if-else layout is a bit clearer to read. If its not, then this PR is kind of pointless :D